resources/authgear/templates/en/translation.json

```diff
@@ -848,7 +848,7 @@
   "v2-signup-login-title": "Welcome",
   "v2-signup-login-subtitle": "Sign up or Log in to {AppName}",
   "v2-login-title": "Welcome",
-  "v2-login-subtitle": "{ClientName, select, null {Login to {AppName}} other {Login to {AppName} to continue to {ClientName}}}",
+  "v2-login-subtitle": "{ClientName, select, null {Login to {AppName}} other {Login to {ClientName}}}",
```

Comment:
Can we change "v2-signup-login-subtitle" and "v2-signup-subtitle" so that it will use ClientName if given too?